addons/website_forum/views/forum_forum_templates_post.xml

```diff
@@ -34,9 +34,11 @@
                     <i class="fa fa-check me-1"/>Solved
             </span>
         </div>
-        <!-- [TODO] Note: after closing and reopening a post, the closed_reason_id remains on post  -->
-        <!-- <t t-out="question.closed_reason_id.name"></t> -->
-        <!-- [TODO] non active posts aren't listed -->
+        <span t-if="question.active" class="text-muted">
+                <t t-if="question.state=='pending'"> [Waiting Validation]</t>
+                <t t-if="question.state=='flagged' and user.karma > forum.karma_flag"> [Flagged]</t>
+                <t t-if="question.state=='close'"> [Closed]</t>
+        </span>
         <span t-if="not question.active" class="text-muted">
             <t t-if="question.state!='offensive'"> [Deleted]</t>
             <t t-if="question.state=='offensive'"> [Offensive]</t>
```

Comment:
```suggestion
        <span class="text-muted">
                <t t-if="question.state=='pending'"> [Waiting Validation]</t>
                <t t-if="question.state=='flagged' and user.karma > forum.karma_flag"> [Flagged]</t>
                <t t-if="question.state=='close'"> [Closed]</t>
                <t t-if="not question.active"> [Deleted]</t>
                <t t-if="question.state=='offensive'"> [Offensive]</t>
```

Reply:
In case I change it the way you proposed, we will get multiple "tags" for each post. Like for example if the post is closed and then someone decides to delete the post, the deleted post will have both [Closed] and [Deleted] next to its title. Do you think it's worth showing the state of the post when the post is already deleted?

Reply:
Sorry, I mean't `t-elif`, and to be optimized. If we don't want multiple tags, start with the active test. Let's not duplicate for now if it isn't already.